index.ts

```diff
@@ -9,24 +9,42 @@ if (MATBAS.build === 'debug') {
 import { Module } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 
-import { NestLogger } from './debug';
-
+import { Exit } from './exit';
 import { Translator } from './translator';
 import { Config, Argv, Env } from './config';
 import { ConsoleHost } from './host';
 import { Editor } from './editor';
 import { Executor } from './executor';
 
+async function exit(code: number) {
+  process.exit(code);
+}
+
+function configFactory(argv: Argv, env: Env) {
+  try {
+    return Config.load(argv, env);
+  } catch (err) {
+    // Normally we would count on the Translator to handle these sorts of
```

Comment:
This has been broken ever since Nest came into the picture. It suggests that there are gaps around interface-level testing. Additionally, there are known gaps with testing for release builds.